[2021-05-26 18:00:01,307] WARN Client session timed out, have not heard from server in 18905ms for sessionid 0x10001b051f50002 (org.apache.zookeeper.ClientCnxn)
[2021-05-26 18:00:01,357] WARN [Controller id=0, targetBrokerId=0] Error connecting to node 192.168.67.7:9092 (id: 0 rack: null) (org.apache.kafka.clients.NetworkClient)
java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:476)
	at java.base/sun.nio.ch.Net.connect(Net.java:468)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:277)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:255)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:987)
	at org.apache.kafka.clients.NetworkClient.ready(NetworkClient.java:311)
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:65)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:292)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:246)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:96)
[2021-05-26 18:00:01,637] INFO Client session timed out, have not heard from server in 18905ms for sessionid 0x10001b051f50002, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
[2021-05-26 18:00:03,831] WARN [Controller id=0, targetBrokerId=0] Error connecting to node 192.168.67.7:9092 (id: 0 rack: null) (org.apache.kafka.clients.NetworkClient)
java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:476)
	at java.base/sun.nio.ch.Net.connect(Net.java:468)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:277)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:255)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:987)
	at org.apache.kafka.clients.NetworkClient.ready(NetworkClient.java:311)
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:65)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:292)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:246)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:96)
[2021-05-26 18:00:03,959] WARN [Controller id=0, targetBrokerId=0] Error connecting to node 192.168.67.7:9092 (id: 0 rack: null) (org.apache.kafka.clients.NetworkClient)
java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:476)
	at java.base/sun.nio.ch.Net.connect(Net.java:468)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:277)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:255)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:987)
	at org.apache.kafka.clients.NetworkClient.ready(NetworkClient.java:311)
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:65)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:292)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:246)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:96)
[2021-05-26 18:00:04,071] WARN [Controller id=0, targetBrokerId=0] Error connecting to node 192.168.67.7:9092 (id: 0 rack: null) (org.apache.kafka.clients.NetworkClient)
java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:476)
	at java.base/sun.nio.ch.Net.connect(Net.java:468)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:277)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:255)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:987)
	at org.apache.kafka.clients.NetworkClient.ready(NetworkClient.java:311)
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:65)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:292)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:246)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:96)
[2021-05-26 18:00:04,184] WARN [Controller id=0, targetBrokerId=0] Error connecting to node 192.168.67.7:9092 (id: 0 rack: null) (org.apache.kafka.clients.NetworkClient)
java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:476)
	at java.base/sun.nio.ch.Net.connect(Net.java:468)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:277)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:255)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:987)
	at org.apache.kafka.clients.NetworkClient.ready(NetworkClient.java:311)
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:65)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:292)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:246)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:96)
[2021-05-26 18:00:04,299] WARN [Controller id=0, targetBrokerId=0] Error connecting to node 192.168.67.7:9092 (id: 0 rack: null) (org.apache.kafka.clients.NetworkClient)
java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:476)
	at java.base/sun.nio.ch.Net.connect(Net.java:468)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:277)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:255)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:987)
	at org.apache.kafka.clients.NetworkClient.ready(NetworkClient.java:311)
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:65)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:292)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:246)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:96)
[2021-05-26 18:00:04,923] WARN [Controller id=0, targetBrokerId=0] Error connecting to node 192.168.67.7:9092 (id: 0 rack: null) (org.apache.kafka.clients.NetworkClient)
java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:476)
	at java.base/sun.nio.ch.Net.connect(Net.java:468)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:277)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:255)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:987)
	at org.apache.kafka.clients.NetworkClient.ready(NetworkClient.java:311)
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:65)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:292)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:246)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:96)
[2021-05-26 18:00:05,046] WARN [Controller id=0, targetBrokerId=0] Error connecting to node 192.168.67.7:9092 (id: 0 rack: null) (org.apache.kafka.clients.NetworkClient)
java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:476)
	at java.base/sun.nio.ch.Net.connect(Net.java:468)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:277)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:255)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:987)
	at org.apache.kafka.clients.NetworkClient.ready(NetworkClient.java:311)
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:65)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:292)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:246)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:96)
[2021-05-26 18:00:05,203] WARN [Controller id=0, targetBrokerId=0] Error connecting to node 192.168.67.7:9092 (id: 0 rack: null) (org.apache.kafka.clients.NetworkClient)
java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:476)
	at java.base/sun.nio.ch.Net.connect(Net.java:468)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:277)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:255)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:987)
	at org.apache.kafka.clients.NetworkClient.ready(NetworkClient.java:311)
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:65)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:292)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:246)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:96)
[2021-05-26 18:00:05,314] WARN [Controller id=0, targetBrokerId=0] Error connecting to node 192.168.67.7:9092 (id: 0 rack: null) (org.apache.kafka.clients.NetworkClient)
java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:476)
	at java.base/sun.nio.ch.Net.connect(Net.java:468)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:277)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:255)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:987)
	at org.apache.kafka.clients.NetworkClient.ready(NetworkClient.java:311)
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:65)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:292)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:246)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:96)
[2021-05-26 18:00:05,425] WARN [Controller id=0, targetBrokerId=0] Error connecting to node 192.168.67.7:9092 (id: 0 rack: null) (org.apache.kafka.clients.NetworkClient)
java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:476)
	at java.base/sun.nio.ch.Net.connect(Net.java:468)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:277)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:255)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:987)
	at org.apache.kafka.clients.NetworkClient.ready(NetworkClient.java:311)
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:65)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:292)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:246)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:96)
[2021-05-26 18:00:05,733] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-05-26 18:00:05,734] INFO Socket connection established, initiating session, client: /127.0.0.1:58539, server: localhost/127.0.0.1:2181 (org.apache.zookeeper.ClientCnxn)
[2021-05-26 18:00:05,799] WARN [Controller id=0, targetBrokerId=0] Error connecting to node 192.168.67.7:9092 (id: 0 rack: null) (org.apache.kafka.clients.NetworkClient)
java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:476)
	at java.base/sun.nio.ch.Net.connect(Net.java:468)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:277)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:255)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:987)
	at org.apache.kafka.clients.NetworkClient.ready(NetworkClient.java:311)
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:65)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:292)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:246)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:96)
[2021-05-26 18:00:05,859] INFO EventThread shut down for session: 0x10001b051f50002 (org.apache.zookeeper.ClientCnxn)
[2021-05-26 18:00:05,809] INFO [ZooKeeperClient Kafka server] Session expired. (kafka.zookeeper.ZooKeeperClient)
[2021-05-26 18:00:05,808] WARN Unable to reconnect to ZooKeeper service, session 0x10001b051f50002 has expired (org.apache.zookeeper.ClientCnxn)
[2021-05-26 18:00:05,975] INFO Unable to reconnect to ZooKeeper service, session 0x10001b051f50002 has expired, closing socket connection (org.apache.zookeeper.ClientCnxn)
[2021-05-26 18:00:06,083] INFO [ZooKeeperClient Kafka server] Initializing a new session to localhost:2181. (kafka.zookeeper.ZooKeeperClient)
[2021-05-26 18:00:06,150] INFO Initiating client connection, connectString=localhost:2181 sessionTimeout=18000 watcher=kafka.zookeeper.ZooKeeperClient$ZooKeeperClientWatcher$@69653e16 (org.apache.zookeeper.ZooKeeper)
[2021-05-26 18:00:06,186] INFO jute.maxbuffer value is 4194304 Bytes (org.apache.zookeeper.ClientCnxnSocket)
[2021-05-26 18:00:06,187] INFO zookeeper.request.timeout value is 0. feature enabled= (org.apache.zookeeper.ClientCnxn)
[2021-05-26 18:00:06,237] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-05-26 18:00:06,238] INFO Socket connection established, initiating session, client: /127.0.0.1:58541, server: localhost/127.0.0.1:2181 (org.apache.zookeeper.ClientCnxn)
[2021-05-26 18:00:06,451] INFO Session establishment complete on server localhost/127.0.0.1:2181, sessionid = 0x10001b051f50003, negotiated timeout = 18000 (org.apache.zookeeper.ClientCnxn)
[2021-05-26 18:00:06,465] INFO Updated cache from existing FinalizedFeaturesAndEpoch(features=Features{}, epoch=0) to latest FinalizedFeaturesAndEpoch(features=Features{}, epoch=0). (kafka.server.FinalizedFeatureCache)
[2021-05-26 18:00:06,494] INFO Creating /brokers/ids/0 (is it secure? false) (kafka.zk.KafkaZkClient)
